packages/esm-ward-app/src/ward-view/ward-view.test.tsx

```diff
@@ -20,10 +27,15 @@ jest.mocked(useSession).mockReturnValue({
   sessionId: 'sessionId',
 });
 
+const mockFeatureFlag: jest.Mock = jest.fn(() => true);
```

Comment:
You should be able to just do `const mockedUseFeatureFlag = useFeatureFlag  as jest.Mock;`.